hadoop-ozone/recon/src/main/java/org/apache/hadoop/ozone/recon/api/PipelineEndpoint.java

```diff
@@ -130,25 +130,55 @@ public Response getPipelines() {
     return Response.ok(pipelinesResponse).build();
   }
 
-  private Optional<Long> getMetricValue(String metricName, String groupId) {
+  private Long getElectionCountMetricValue(String groupId) {
+    Long electionCount = 0L;
     String metricsQuery = String.format(
-        "query=%s{group=\"%s\"}", metricName, groupId);
+            "query=ratis_leader_election_electionCount{group=\"%s\"}", groupId);
     try {
       List<Metric> metrics = metricsServiceProvider.getMetricsInstant(
-          metricsQuery);
+              metricsQuery);
+      if (!metrics.isEmpty()) {
+        for (Metric m : metrics) {
+          TreeMap<Double, Double> values = (TreeMap<Double, Double>)
+                  m.getValues();
+          if (!values.isEmpty()) {
+            electionCount += Optional.of(values.firstEntry().getValue()
+                    .longValue()).orElse(0L);
+          }
+        }
+      }
+    } catch (Exception ex) {
```

Comment:
I wanted to say we preferred catching a more detailed exception than such a generic one, but I figured it is not easy to do so at the moment because `getMetricsInstant()` throws generic exception atm.